### DISCUSSION TITLE
Long loading time with storage

### DISCUSSION THREAD

**Discussion by krzar** [GrapesJS/grapesjs#3922] [Q&A]
I load data from Storage Manager my html code contains two extended SVG images making the editor take a very long time to load everything, up to a minute. Is it possible for grapejs to skip the svg tags?

It unnecessarily adds `data-gjs-type="svg" `to all tags, while I only want to display those svg.

**Comment by artf** [comment]
You can load the SVG as an image but sounds weird anyway that is taking so long (it might be your server if you're using the remote storage). Create a simple demo so we can check what is going on.

**Reply by krzar** [reply]
There are two large svg files, each about 11k lines long. They need to be inline because there are click and hover events. 
After commenting out the svg everything works fine. Is there no way for grapejs to skip tags?
